header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -449,6 +451,69 @@ public function define_row_settings( $row_id ) {
 				'default'               => $default_colors['text'],
 			]
 		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                    => self::BOTTOM_BORDER,
+				'group'                 => $row_setting_id,
+				'tab'                   => SettingsManager::TAB_STYLE,
+				'section'               => $row_setting_id,
+				'label'                 => __( 'Divider width', 'neve' ),
+				'type'                  => '\Neve\Customizer\Controls\React\Responsive_Range',
+				'live_refresh_selector' => $row_class,
+				'live_refresh_css_prop' => [
+					'cssVar' => [
+						'responsive' => true,
+						'vars'       => '--headerBorderWidth',
+						'suffix'     => 'px',
+						'fallback'   => '0',
+						'selector'   => '.' . $this->get_id() . '-' . $row_id,
+					],
+					'prop'   => 'border-bottom-width',
+					'unit'   => 'px',
+				],
+				'options'               => [
+					'input_attrs' => [
+						'step'       => 1,
+						'min'        => 0,
+						'max'        => 50,
+						'defaultVal' => [
+							'mobile'  => 0,
+							'tablet'  => 0,
+							'desktop' => 0,
+						],
+						'units'      => [ 'px' ],
+					],
+				],
+				'transport'             => 'postMessage',
+				'sanitize_callback'     => array( $this, 'sanitize_responsive_int_json' ),
+				'default'               => '{ "mobile": "0", "tablet": "0", "desktop": "0" }',
+				'conditional_header'    => $this->get_id() === 'header',
+			]
+		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                    => self::BORDER_COLOR,
+				'group'                 => $row_setting_id,
+				'tab'                   => SettingsManager::TAB_STYLE,
+				'label'                 => __( 'Divider Color', 'neve' ),
+				'section'               => $row_setting_id,
+				'conditional_header'    => $this->get_id() === 'header',
+				'type'                  => 'neve_color_control',
+				'transport'             => 'postMessage',
+				'live_refresh_selector' => $row_class,
+				'live_refresh_css_prop' => [
+					'cssVar' => [
+						'vars'     => '--headerBorderColor',
+						'selector' => '.' . $this->get_id() . '-' . $row_id,
+					],
+				],
+				'sanitize_callback'     => 'wp_filter_nohtml_kses',
```

Comment:
We should use[ `neve_sanitize_colors`](https://github.com/codeinwp/neve/blob/02f565072c424201b460295f3997b231f9992a75/globals/sanitize-functions.php#L19-L34) for the sanitize callback on color controls 🦺